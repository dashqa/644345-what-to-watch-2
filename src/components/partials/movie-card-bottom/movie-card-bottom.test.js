import React from 'react';
import renderer from 'react-test-renderer';
import MovieCardBottom from './movie-card-bottom';

const props = {
  movie: {
    name: ``,
    genre: ``,
    released: 0,
    backgroundImage: ``,
    posterImage: ``,
    runTime: 0,
    director: ``,
    starring: [],
  },
};

it(`Movie card bottom correctly renders after relaunch`, () => {
  const tree = renderer.create(<MovieCardBottom {...props}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
