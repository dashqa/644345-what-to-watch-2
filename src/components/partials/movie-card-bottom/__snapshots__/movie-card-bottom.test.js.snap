// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie card bottom correctly renders after relaunch 1`] = `
<div
  className="movie-card__wrap movie-card__translate-top"
>
  <div
    className="movie-card__info"
  >
    <div
      className="movie-card__poster movie-card__poster--big"
    >
      <img
        alt=""
        height="327"
        src="/"
        width="218"
      />
    </div>
    <div
      className="movie-card__desc"
    >
      <nav
        className="movie-nav movie-card__nav"
      >
        <ul
          className="movie-nav__list"
        >
          <li
            className="movie-nav__item"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Overview
            </a>
          </li>
          <li
            className="movie-nav__item movie-nav__item--active"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Details
            </a>
          </li>
          <li
            className="movie-nav__item"
          >
            <a
              className="movie-nav__link"
              href="#"
            >
              Reviews
            </a>
          </li>
        </ul>
      </nav>
      <div
        className="movie-card__text movie-card__row"
      >
        <div
          className="movie-card__text-col"
        >
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Director
            </strong>
            <span
              className="movie-card__details-value"
            >
              
            </span>
          </p>
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Starring
            </strong>
            <span
              className="movie-card__details-value"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "",
                }
              }
            />
          </p>
        </div>
        <div
          className="movie-card__text-col"
        >
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Run Time
            </strong>
            <span
              className="movie-card__details-value"
            >
              0
            </span>
          </p>
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Genre
            </strong>
            <span
              className="movie-card__details-value"
            >
              
            </span>
          </p>
          <p
            className="movie-card__details-item"
          >
            <strong
              className="movie-card__details-name"
            >
              Released
            </strong>
            <span
              className="movie-card__details-value"
            >
              0
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
